cmd/oci-runtime-tool/generate.go

```diff
@@ -740,6 +744,21 @@ func readKVStrings(files []string, override []string) ([]string, error) {
 	return envVariables, nil
 }
 
+// parseEnv splits a given environment variable (of the form name=value) into
+// (name, value). An error is returned if there is no "=" in the line.
```

Comment:
I think comment here is not suitable,  also need to be updated. But it's tiny problem.